qa/SidechainTestFramework/account/smart_contract_resources/contracts/ForgerStakes.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Actually this is exactly the same file as `./qa/truffle/contracts/ForgerStakes.sol`, which is currently useful only as reference for manual tests to be executed outside the test framework. Consider removing it.